### ISSUE TITLE
[FEATURE] Add helpful error messages

### ISSUE THREAD

**Issue by oliverklee** [TYPO3/tailor#52]
## Summary / Description
Currently, my TER release is failing with the following message: 
`No or invalid ext_emconf.php found in the folder.  `

I'd like to have specific error messages like this so I know what the problem is:

- File  `ext_emconf.php` missing
- `$EM_CONF` not set
- `$EM_CONF` is no array
- version info missing in `$EM_CONF`
- typo3 dependency missing in `$EM_CONF`
- version number 1.2.3 in `$EM_CONF` is different than the version number 1.2.4 provided as an argument

**Comment by o-ba** [comment]
Hi, this sound reasonable. Do you want to create a PR for it?

**Comment by oliverklee** [comment]
Yes, will do. (It might take a few days, though.)

**Comment by o-ba** [comment]
Hi @oliverklee, do you still plan to add a PR for this issue?

**Comment by oliverklee** [comment]
Yes. I hope that I’ll get it done during the T3DD22 coding night.

**Comment by josefglatz** [comment]
Any update on this?

**Comment by oliverklee** [comment]
No, unfortunately not, and I currently don't know yet when I'll have time for this. Sorry.

If anyone would be willing to step in, that would help.

**Comment by eliashaeussler** [comment]
@oliverklee You may want to check out #78 😉 I hope this covers most of your cases mentioned here.
